test/e2e/reporters/progress.go

```diff
@@ -33,6 +33,11 @@ import (
 // ProgressReporter is a ginkgo reporter which tracks the total number of tests to be run/passed/failed/skipped.
 // As new tests are completed it updates the values and prints them to stdout and optionally, sends the updates
 // to the configured URL.
+
+// One known limitation of the ProgressReporter it this reporter will not consolidate the reporter from each sub-process
+// if the tests are run in parallel.
+// As what's observed the reporter sent before test suite is started will be assembled correctly but each process will report
+// on its own after each test or suite are completed.
```

Comment:
Perhaps include the message from your TCP dump as example here?

Reply:
Done